README.md

```diff
@@ -1,3 +1,22 @@
+# Spectrometer
+
+Spectrometer is a polyglot dependency analysis tool, designed to identify and
+report dependency trees for many languages and package managers.  See the
+[docs](docs/strategies.md) for a non-exhaustive list of supported
+languages/managers.
+
+```sh
+me@mydesk:~/myproject$ fossa2 analyze -o
+# Dependency analysis output
+```
+
+## Table of Contents
+
+1. [Installation](#installation)
+1. [Basic Usage](#basic-usage)
+3. [Supported Strategies](#supported-languagesmanagers)
+4. [Contributing/Building From Source](#contributingbuilding-from-source)
+
 ## Installation
```

Comment:
Do we have automated testing in this repository for the installation commands yet?

Reply:
Not likely.

Reply:
We don't. As soon as we can dogfood with a haskell analyzer, it'll be part of the test suite (for at least the linux installer)